iml-system-test-utils/src/snapshots.rs

```diff
@@ -67,46 +66,16 @@ pub enum SnapshotName {
     StratagemDetected,
 }
 
-impl SnapshotName {
```

Comment:
The ordering isn't needed?

Reply:
The ordering is still there, it's just derived by the compiler instead of us having to write it by hand.